### PR TITLE
Ria 3309 ho hearing

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3309

### Change description ###
RIA-3309 ho hearing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-prepareforhearing.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        "listCaseHearingDate": "2018-12-31T12:34:56",
+        "listCaseRequirementsVulnerabilities": "something around Vulnerabilities",
+        "listCaseRequirementsMultimedia": "something around Multimedia",
+        "listCaseRequirementsSingleSexCourt": "something around SingleSexCourt",
+        "listCaseRequirementsInCameraCourt": "something around InCameraCourt",
+        "listCaseRequirementsOther": "something around Other",
+        "uploadAdditionalEvidenceActionAvailable": "Yes",
+        "ariaListingReference": "LP/12345/2019",
+        "decisionHearingFeeOption": "decisionWithHearing",
+        "homeOfficeHearingInstructStatus": "OK"
```
Comment (jkumar-moj) [inline src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-prepareforhearing.json:77]: I mentioned earlier about having a separare flag. do you think it is not needed?

File: src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-decision.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{
+  "description": "RIA-3309 sends ho notification for edit case listing event for state decision",
+  "enabled": "{$featureFlag.isHomeOfficeIntegrationEnabled}",
```
[the remark this answers is not in the document]
Comment (Lee365) [reply src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-decision.json]: Added

File: src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-prehearing.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
+{
+  "description": "RIA-3309 sends ho notification for edit case listing event for state preHearing",
+  "enabled": "{$featureFlag.isHomeOfficeIntegrationEnabled}",
```
[the remark this answers is not in the document]
Comment (Lee365) [reply src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-prehearing.json]: Added

File: src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-prepareforhearing.json (hunk 1 above)
Comment (Lee365) [reply src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-prepareforhearing.json:77]: No, I don't think there is a need for separate flag.

File: src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-finalbundling.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
+{
+  "description": "RIA-3309 sends ho notification for edit case listing event for state finalBundling",
+  "enabled": "{$featureFlag.isHomeOfficeIntegrationEnabled}",
```
[the remark this answers is not in the document]
Comment (Lee365) [reply src/functionalTest/resources/scenarios/RIA-3309-ho-notifications-for-edit-case-listing-finalbundling.json]: Added

**Review by jkumar-moj — APPROVED** [review]
👍

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**